_posts/en/newsletters/2022-06-15-newsletter.md

```diff
@@ -0,0 +1,286 @@
+      want, wasting bandwidth.  BIP author Gloria Zhao [suggests][zhao
+      negotiation] using that the first version of the package relay
+      protocol should imply a maximum package size of 25 transactions
+      and 101,000 vbytes.
+
+    - *Package graph announcement only:* Eric Voskuil
+      [recommends][voskuil graph] that a peer who learns about a
+      high-feerate descendant of a low-feerate ancestor should simply
+      inform its peers of the relationships between those two
+      transactions, called the package graph.  A receiving peer can then
+      request any transactions it doesn't have.  In a separate part of
+      the thread, Towns [notes][towns graph] that a graph can't be
+      validated until all transactions have been received, so care must
+      be taken to ensure a peer can't lie about a graph in order to
+      prevent a transaction from being relayed by other peers.
+
+    - *Using short ids:* several developers suggested using
+      [BIP152][]-style short identifiers (ids).  Zhao [explained][zhao
+      sids] that short ids make sense for block relay where nodes first
+      validate a new block's proof of work (which is expensive to
+      create), so it would be expensive for an attacker to abuse the
+      mechanism to waste node resources.  But, for relay where no
+      significant expense is involved, short ids can be abused over and
+      over again to potentially create a denial-of-service attack.
+
+    - *Non-standard parents:* Suhas Daftuar [describes][daftuar repeat]
+      a scenario where a node implementing package relay can end up
+      repeatedly requesting the same package.  This is would be
+      especially likely to happen when relay policy differs between
+      older and newer nodes, such as after a soft fork is activated.
+
+    - *Challenges of a block hash beacon:* Daftuar also notes that one
+      feature of the proposal may cause problems for other software.
+      The current draft BIP has packages include the node's current hash
+      of the latest block on the block chain.  This allows the receiving
+      peer to ignore a package if it's from an earlier block (or
+      alternative chain), in which case the package may not work with
+      the receiving peer's current mempool.  However, Daftuar notes that
+      there's probably a lot of software that sends transactions---and
+      which may eventually like to send packages---which doesn't keep
+      track of the current chain tip hash.
+
+- **Summary of LN developer meeting:** Olaoluwa Osuntokun provided a
+  [detailed summary][osuntokun summary] of the LN dev meeting in Oakland
+  last week.  Topics covered included:
+
+    - *Taproot-based LN channels:* the first step of moving LN to full
+      use of [taproot's][topic taproot] features, with later steps
+      likely to include support for [PTLCs][topic ptlc] (see also
+      [Newsletter #164][news164 taproot ln]).
```

Comment:
This paragraph feels incomplete. Is the first step of moving LN to Taproot taproot-based channels or is this just a setup for the next paragraphs? I assume this means using a P2TR output in the commitment transaction. Perhaps that could be clarified.